plugins/modules/mount.py

```diff
@@ -797,7 +870,7 @@ def main():
                         msg="Error unmounting %s: %s" % (name, msg))
 
             changed = True
-    elif state == 'mounted':
+    elif state == 'mounted' or state == 'ephemeral':
```

Comment:
`ephemeral` treatment is based on the `mounted` logic, bypassing the *fstab* read and write.